src/types/Filter.ts

```diff
@@ -0,0 +1 @@
+export type Filter = 'all' | 'active' | 'completed';
```

Comment:
```suggestion
export type Filters = 'all' | 'active' | 'completed';
```
I think `Filters` would be little be more explicit - we have more than one filter, but that's only a suggestion